### PR TITLE
[Windows] Move Windows features to toolset

### PR DESCRIPTION
**Pull request by maxim-lobanov**
# Description
This PR moves the list of windows features to be activated to toolset to simplify future changes.
Also implements a few minor improvements in other code:
- Move `Fix-ImportPublishProfile` logic to the separate function
- Remove extra debug logs from PostgreSQL installation

## Check list
- [ ] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by al-cheb — APPROVED** [review]

**Review by MaksimZhukov — APPROVED** [review]

**Review by AlenaSviridenko — APPROVED** [review]

File: images/win/scripts/Installers/Install-DotnetSDK.ps1 [hunk 1]
```diff
@@ -29,6 +29,18 @@ function Invoke-Warmup (
...
+    if (Test-IsWin16 -or Test-IsWin19) {
+        # Fix for issue https://github.com/dotnet/sdk/issues/1276.  This will be fixed in 3.1.
+        $sdkTargetsName = "Microsoft.NET.Sdk.ImportPublishProfile.targets"
+        $sdkTargetsUrl = "https://raw.githubusercontent.com/dotnet/sdk/82bc30c99f1325dfaa7ad450be96857a4fca2845/src/Tasks/Microsoft.NET.Build.Tasks/targets/${sdkTargetsName}"
+        $sdkTargetsPath = "C:\Program Files\dotnet\sdk\$sdkVersion\Sdks\Microsoft.NET.Sdk\targets"
+        Start-DownloadWithRetry -Url $sdkTargetsUrl -DownloadPath $sdkTargetsPath -Name $sdkTargetsName
+    }
+}
+
 function InstallSDKVersion (
```
[the remark this answers is not in the document]
Comment (maxim-lobanov) [reply images/win/scripts/Installers/Install-DotnetSDK.ps1:44]: I would avoid modifying code that is not directly related to my changes.

**Pull request merged**